modules/openapi-generator/src/main/resources/python/python-experimental/model_test.mustache

```diff
@@ -30,4 +30,4 @@ class Test{{unescapedDescription}}(unittest.TestCase):
 {{/models}}
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main()
```

Comment:
Can you add the newline back to this file?
It doesn't look like we need to edit this file.

Reply:
Done